Makefile

```diff
@@ -75,19 +76,22 @@ deploy:
 	$(MAKE) -C secret-provider deploy
 	$(MAKE) -C manager deploy
 	$(MAKE) -C connectors deploy
+	$(MAKE) -C connectors/katalog deploy
```

Comment:
No such target. Exclude deployment from this PR because it will be part of the chart.